ledger/store/trackerdb/sqlitedriver/schema.go

```diff
@@ -344,6 +344,36 @@ func accountsCreateBoxTable(ctx context.Context, tx *sql.Tx) error {
 	return nil
 }
 
+// performKVStoreNullBlobConversion scans keys with null blob value, and convert the value to `[]byte{}`.
+func performKVStoreNullBlobConversion(ctx context.Context, tx *sql.Tx) error {
+	updateWithEmptyByteSlice, err := tx.PrepareContext(ctx, "UPDATE kvstore SET value = ? WHERE key = ?")
+	if err != nil {
+		return err
+	}
+
+	var rows *sql.Rows
+	rows, err = tx.QueryContext(ctx, "SELECT key FROM kvstore WHERE value is NULL;")
+	if err != nil {
+		return err
+	}
+
+	var boxKey []byte
+
+	for rows.Next() {
+		if err = rows.Scan(&boxKey); err != nil {
+			rows.Close()
+			return err
+		}
+		_, err = updateWithEmptyByteSlice.ExecContext(ctx, []byte{}, boxKey)
```

Comment:
Can we perform the entire schema migrate with:
tx.QueryContext(ctx, "UPDATE kvstore SET value = "" WHERE value is NULL;")

Reply:
right, even better! see [trackerdb migration step conversion](https://github.com/algorand/go-algorand/pull/5225/commits/f9b7713fd86f9df84b9daf790419e83f74ca6ff1)

Reply:
even even better! see [empty byte slice literal representation in sql query](https://github.com/algorand/go-algorand/pull/5225/commits/9f53546012e98daeb3c84d3c2f25a89fdba67aa4), we have the empty byte slice representation get into sql query, so no `[]byte{}` needed

Reply:
Ok, so the last thing I want to check is whether we've fixed both write paths.  This PR fixes the write path during fast catchup.  Have we also ensured that the "normal" write path, that occurs after an AVM program executes:
```
byte "hello"
int 0
box_create
```
also correctly writes a '', rather than a `nil`?

Reply:
local produce seems it works